packages/core/src/util/normalizeConfig.js

```diff
@@ -511,37 +517,13 @@ function normalizeNumericalStackTraceLength(numericalLength) {
 /**
  * @param {InstanaConfig} config
  */
-function normalizeDisabledTracers(config) {
-  if (
-    config.tracing.disabledTracers == null &&
-    process.env['INSTANA_DISABLED_TRACERS'] &&
-    process.env['INSTANA_DISABLED_TRACERS'].trim().length >= 0
-  ) {
-    config.tracing.disabledTracers = process.env['INSTANA_DISABLED_TRACERS']
-      .split(',')
-      .map(key => key.trim().toLowerCase())
-      .filter(key => key.length >= 0);
-  }
-
-  if (!config.tracing.disabledTracers) {
-    config.tracing.disabledTracers = defaults.tracing.disabledTracers;
-  }
-
-  if (!Array.isArray(config.tracing.disabledTracers)) {
-    logger.warn(
-      `Invalid configuration: config.tracing.disabledTracers is not an array, the value will be ignored: ${JSON.stringify(
-        config.tracing.disabledTracers
-      )}`
-    );
-    config.tracing.disabledTracers = defaults.tracing.disabledTracers;
+function normalizeDisableTracers(config) {
+  const disableConfig = configNormalizers.disable.normalize(config);
+  if (disableConfig.libraries && disableConfig.libraries.length > 0) {
+    config.tracing.disable = configNormalizers.disable.normalize(config);
```

Comment:
Can you please explain why we call `configNormalizers.disable.normalize` twice? 🤔